cou/utils/text_styler.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2023 Canonical Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Command line text styling utilities."""
+
+from typing import Optional
+
+from colorama import Style
+
+
+def bold(text: str) -> str:
+    """Transform the text in bold format.
+
+    :param text: text to format.
+    :type text: str
+    :return: text formatted.
+    :rtype: str
+    """
+    return Style.RESET_ALL + Style.BRIGHT + text + Style.RESET_ALL
+
+
+def normal(text: str) -> str:
+    """Transform the text in normal format.
+
+    :param text: text to format.
+    :type text: str
+    :return: text formatted.
+    :rtype: str
+    """
+    return Style.RESET_ALL + text + Style.RESET_ALL
+
+
+def prompt_message(parameter: str, default_choice: Optional[str] = None) -> str:
+    """Generate eye-catching prompt.
+
+    :param parameter: String to show at the prompt with the user options.
+    :type parameter: str
+    :param default_choice: Default choice if user doesn't a provide valid input.
+    :type default_choice: Optional[str]
+    :return: Prompt string with the user options.
+    :rtype: str
+    :raise ValueError: raise ValueError if default choice is invalid
+    """
+    continue_option = "y"
+    abort_option = "n"
+
+    if not default_choice:  # use all lowercases if no default is passed
+        pass
+    elif default_choice.casefold() == "y":
+        continue_option = "Y"
```

Comment:
Because when "y" is the default choice, we should display it in upper case

Reply:
So why not simply use `default_choice.upper()`

Reply:
Because we need to check whether the `default_choice` is valid, and also 'y' and 'n' are in different position of the message so we need to know which one needs to be in uppercase and which one should remain lower.

Reply:
I do not think we need to check if default value is valid, since those values are cumming from source code and not from any config, user inputs, etc.

Reply:
To me I prefer having an exception here even if it only been used internally. This makes the program more robust.

Reply:
This make program being same, but more robust to developer error. We spent some time to discussion it and yet do not see that the default value is not used if user pressed only enter.
IMO instead of validating potential developer error, we can design it to have single function handling the input and provide the output.